plugins/node/opentelemetry-instrumentation-memcached/src/instrumentation.ts

```diff
@@ -138,7 +138,7 @@ export class Instrumentation extends InstrumentationBase<typeof Memcached> {
       const query = original.apply(this, arguments as any);
       const callback = query.callback;
 
-      span.updateName(`${query.type} ${query.key}`);
+      span.updateName(`memcached ${query.type}`);
```

Comment:
[there are tests](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/4c8843be14896d1159a622c07eb3a049401ccba1/plugins/node/opentelemetry-instrumentation-memcached/test/index.test.ts#L283) asserting the key is found in the span name. They should be updated as well

Reply:
This is done.